remo/profiles/forms.py

```diff
@@ -220,11 +226,20 @@ def clean(self):
             return cdata
 
         tomorrow = get_date(days=1)
-        if 'expected_date' in cdata and cdata['expected_date'] < tomorrow:
-            msg = ('Return day cannot be earlier than {return_day}'
-                   .format(return_day=tomorrow.strftime('%d %B %Y')))
-            self._errors['expected_date'] = self.error_class([msg])
-            del cdata['expected_date']
+        max_period = get_date(weeks=MAX_UNAVAILABILITY_PERIOD)
+        if 'expected_date' in cdata:
+            if cdata['expected_date'] < tomorrow:
+                msg = ('Return day cannot be earlier than {return_day}'
+                       .format(return_day=tomorrow.strftime('%d %B %Y')))
+                self._errors['expected_date'] = self.error_class([msg])
+                del cdata['expected_date']
+            if cdata['expected_date'] > max_period:
+                msg = (u'The maximum period for unavailability is until {0}.'
```

Comment:
I suggest we keep the same format in string formatting. Use either ``0`` or descriptive string.